packages/node/src/cli.js

```diff
@@ -378,10 +401,17 @@ const main = async (args = hideBin(process.argv)) => {
           /**
            * Resolve entrypoint from `root`
            */
-          .middleware(
+          .middleware([
             processEntrypointMiddleware,
-            true // RUN BEFORE CHECK FN
-          ),
+            (argv) => {
```

Comment:
- [ ] verify that we still need this; I think it may be handled fully by `loadPolicies()`

Reply:
not needed; I removed it.